clients/hadoop-to-venice-bridge/src/main/java/com/linkedin/venice/hadoop/input/kafka/ttl/VeniceKafkaInputTTLFilter.java

```diff
@@ -0,0 +1,29 @@
+package com.linkedin.venice.hadoop.input.kafka.ttl;
+
+import com.linkedin.venice.hadoop.AbstractVeniceFilter;
+import com.linkedin.venice.hadoop.VenicePushJob;
+import com.linkedin.venice.hadoop.input.kafka.avro.KafkaInputMapperValue;
+import com.linkedin.venice.utils.VeniceProperties;
+
+
+public class VeniceKafkaInputTTLFilter extends AbstractVeniceFilter<KafkaInputMapperValue> {
+  private final TTLResolutionPolicy ttlPolicy;
+
+  public VeniceKafkaInputTTLFilter(VeniceProperties props) {
+    super(props);
+    ttlPolicy = TTLResolutionPolicy.valueOf(props.getInt(VenicePushJob.REPUSH_TTL_POLICY));
+  }
+
+  @Override
+  protected boolean process(final KafkaInputMapperValue kafkaInputMapperValue) {
+    switch (ttlPolicy) {
+      case REJECT_BATCH_WRITE:
+        // TODO implementation to retrieve RMD and parse timestamp information. See VENG-9956
+        return false; // do not drop anything
```

Comment:
p process returning `false` meaning its not filtered? or rename to `filter`. generally process  means allow to the next stage, ie reducer in this case.

Reply:
Ack. I felt it can be confusing if we have `filter.filter(record)`. I can rename this to `doFilter`

Reply:
may be `apply` or `applyFilter` ?

Reply:
I thought about `handle`, `process` but forget we can have `apply` :smile: